integrated_channels/blackboard/exporters/learner_data.py

```diff
@@ -50,9 +50,14 @@ def get_learner_data_records(
             'blackboard',
             'BlackboardLearnerDataTransmissionAudit'
         )
-
-        return [
-            BlackboardLearnerDataTransmissionAudit(
+        course_id = get_course_id_for_enrollment(enterprise_enrollment)
+        # We only want to send one record per enrollment and course, so we check if one exists first.
+        learner_transmission_record = BlackboardLearnerDataTransmissionAudit.objects.filter(
```

Comment:
How about using get_or_create instead of filter.

Reply:
The problem is that we're not creating the record at this point but creating its model instance. As far as I know, the `get_or_create` method creates the record in db. Correct me if I'm wrong because this method can make the code a lot cleaner.